### PR TITLE
report error for missing closing tag token

### PR DESCRIPTION
**Pull request by emi2k01**
image

As you can see, I used `abort_call_site` instead of adding a more appropriate span like the rustc one:

image

The reason is that I don't know how.

### CODE REVIEW

**Comment by emi2k01** [comment]
I don't know how to make this a draft. The test failing is named "element-missing-body" but HTML tags are not required to have a body, I think.

Should I rename the test to `non-closed-element`?

File: maud_macros/src/parse.rs [hunk 1]
```diff
@@ -553,6 +553,7 @@ impl Parser {
                     );
                 }
             },
+            None => abort_call_site!("expected ';' or '/', found end of macro"),
```
Comment (lambda-fairy) [inline maud_macros/src/parse.rs:556]: I'd rather only mention `;`, as the `/` syntax is soft-deprecated.

Also, Rust style is to use backticks to delimit code snippets.

```suggestion
            None => abort_call_site!("expected `;`, found end of macro"),
```

**Review by lambda-fairy — CHANGES_REQUESTED** [review]
Thanks @emi2k01! Happy to merge once the comment is addressed (and the tests pass).

> Should I rename the test to `non-closed-element`?

Sure, do it!

**Comment by emi2k01** [comment]
Btw, do you know if it's possible to create a diagnostic with whose span is just after the last token? Just like the second screenshot I added to the original comment.

**Review by lambda-fairy — APPROVED** [review]

**Comment by lambda-fairy** [comment]
Thanks!

> Btw, do you know if it's possible to create a diagnostic with whose span is just after the last token?

There is `Span::after`, but it's unstable.

**Pull request merged**